libs/sdk-backend-spi/src/workspace/insights/index.ts

```diff
@@ -115,6 +122,11 @@ export interface IInsightReferences {
      * If requested, metadata about data sets from which this insight queries data will be returned here.
      */
     dataSetMeta?: IMetadataObject[];
+
+    /**
+     * If requested, metadata about analytical dashboards from which this insight queries data will be returned here.
+     */
+    analyticalDashboards?: IMetadataObject[];
```

Comment:
This is not the place to put it, sorry if my previous comments were ambiguous. We should create a new type for the getObjectsReferencing function result.

Reply:
fixed